substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/JfrType.java

```diff
@@ -44,6 +44,8 @@ public enum JfrType {
     FrameType("jdk.types.FrameType"),
     GCCause("jdk.types.GCCause"),
     GCName("jdk.types.GCName"),
+    GCWhen("jdk.types.GCWhen"),
+    VirtualSpace("jdk.types.VirtualSpace"),
```

Comment:
Is this used anywhere?

Reply:
Indeed, it does seem unnecessary. Initially, I added it without fully understanding it. I will remove it and verify that it does not affect the functionality.

Reply:
I removed Definition of Virtual Space from JrfType.